client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -33,6 +34,38 @@ import QRCode from './QRCode'
 import Prose from './Prose'
 import Text from './Text'
 
+const qrCodeTargetClassName = "qr-code-target"
+
+class PagedQRCodeInjector extends Handler {
+  afterRendered(pages: any) {
+    pages.forEach((page: any) => {
+      const { pageNumber } = page.element.dataset
```

Comment:
can't seem to easily find this in the documentation but... you're sure this is 1-indexed?

Reply:
Yessir…

![image](https://user-images.githubusercontent.com/28444/84707678-fbaed580-af13-11ea-88f4-fee494e101d0.png)